roles/pulp3-postgresql/meta/main.yml

```diff
@@ -10,9 +10,9 @@ galaxy_info:
     - name: Fedora
       versions:
         - all
-    - name: Debian
-      versions:
-        - stretch
+    # - name: Debian
+    #   versions:
+    #     - stretch
```

Comment:
Debian support was added but it won't work until the next version of Debian, because Debian Stretch ships with Python 3.5.3, and Pulp only supports >=3.6